<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
    <widget class="QPushButton" name="Btn_Right">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>65</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Left">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Down">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>65</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Up">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Enter">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>ENTER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Esc">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_SendPic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>125</y>
       <width>95</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>加载图片</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>140</y>
       <width>105</width>
       <height>10</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="Lab_Sta">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>125</y>
       <width>105</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>点击发送图片</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Config：8-N-1;COM STA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lab_ComSta">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>155</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action9600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9600</string>
   </property>
  </action>
  <action name="action115200">
   <property name="text">
    <string>115200</string>
   </property>
  </action>
  <action name="action8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>1.5</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
